frontstage/controllers/party_controller.py

```diff
@@ -154,3 +175,25 @@ def verify_token(token):
         raise ApiError(logger, response, log_level=log_level, message='Failed to verify token', token=token)
 
     logger.debug('Successfully verified token')
+
+
+def get_enrolment_details(business_party_id, enrolments):
+    business_details = get_party_by_business_id(business_party_id)
+    enrolments_with_details = [{"business_party": business_details, "enrolment_details": enrolment}
+                               for enrolment in enrolments
+                               if enrolment['enrolmentStatus'] == 'ENABLED']
+
+    return enrolments_with_details
+
+
+def get_survey_list_details_for_party(party_id, tag):
```

Comment:
Tiny thing might not be relevant. Since this function calls the one above it I usually would prefer to have this one on top. Same with the similar thing in collection exercise service